src/pages/academy/components/search_banner/_search-nav.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from 'gatsby'
+import { DesktopWrapper, HoverChevron, StyledChevron } from './_search-styled'
+import SearchBar from './_search-bar'
+import { Flex } from 'components/containers'
+import device from 'themes/device'
+import { Header } from 'components/elements'
+import AcademyLogo from 'images/svg/blog/academy-text.svg'
+
+type NavProps = {
+    setModal?: React.Dispatch<React.SetStateAction<boolean>>
+    openModal: () => void
+    Chevron: string
+    modal_opened: boolean
```

Comment:
We have a role in the deriv-app project: Those variables which store a boolean value, should start with `is_`, `has_`, ...
I don't know we have it in deriv.com or not but base on this role we can change this to `is_modal_opened`

Reply:
@niloo-fs  that convention is also applied in deriv.com.

Reply:
Thank u @prince-deriv